tests/stream/stream_api_fp16.cpp

```diff
@@ -30,7 +30,7 @@ using namespace sycl_cts;
 class test_kernel;
```

Comment:
Should we update the copyright header in this file as well?

Reply:
Right, I forgot about that. Since `stream_api_fp64.cpp` had no license header, I added the one from `docs/README.adoc` with year 2023.